src/explore-education-statistics-admin/src/pages/publication/PublicationDetailsPage.tsx

```diff
@@ -15,39 +16,29 @@ import React from 'react';
 const PublicationDetailsPage = () => {
   const { publication, onReload } = usePublicationContext();
   const {
-    contact,
     id,
-    permissions,
-    supersededById,
-    themeId,
     title,
     summary,
-    topicId,
-  } = publication;
+    permissions,
+    supersededById,
+    theme,
+    topic,
+  } = publication as Publication;
   const [readOnly, toggleReadOnly] = useToggle(true);
 
-  const { value, isLoading } = useAsyncHandledRetry(async () => {
-    const theme = await themeService.getTheme(themeId);
-    const topic = theme?.topics.find(themeTopic => themeTopic.id === topicId);
-    if (!supersededById) {
-      return { theme, topic };
-    }
+  const { value: supersedingPublication, isLoading } = useAsyncHandledRetry(
+    async () => {
+      if (!supersededById) {
+        return undefined;
+      }
 
-    const supersedingPublication = await publicationService.getPublication(id);
-
-    return {
-      theme,
-      topic,
-      supersedingPublication,
-    };
-  });
-
-  const { theme, topic, supersedingPublication } = value ?? {};
+      return publicationService.getPublication(supersededById);
+    },
+  );
```

Comment:
Does `supersededById` need to be added to a dependency array here in `useAsyncHandledRetry`?  Not 100% sure if it needs it in the same way that useEffect() needs it, but would suggest just looking to see if there are any warnings about this in the console when on this page.

Reply:
I've added it to the deps - other places we use `useAsyncHandledRetry` do include variables used in side in the deps.